api/example_docs/m_15_16.xml

```diff
@@ -0,0 +1,367 @@
+<policy emblem="1">
+  <preamble>
+    <published>July 9, 2015</published>
+    <policyNum>M-15-16</policyNum>
+    <policyTitle>
+      MEMORANDUM FOR HEADS OF EXECUTIVE DEPARTMENTS AND AGENCIES
+    </policyTitle>
+    <from marker="FROM:">Shaun Donovan, Director, Office of Management and Budget</from>
+    <from marker="FROM:">Dr. John P. Holdren, Director, Office of Science and Technology Policy</from>
+    <subject marker="SUBJECT:">Multi-Agency Science and Technology Priorities for the FY 2017 Budget</subject>
+  </preamble>
+  <sec>
+    <para>
+      <content>
+        Scientific discovery, technological breakthroughs, and innovation are
+        the primary engines for expanding the frontiers of human knowledge and
+        are vital for responding to the challenges and opportunities ofthe
+        2P1 century.  The Nation depends
+        on science, technology, and innovation to promote economic growth and
+        job creation, maintain a safe and sufficient food supply, improve the
+        health of Americans, move toward a clean energy future, address global
+        climate change, manage competing demands on environmental resources,
+        and ensure the Nation's security. 
+      </content>
```

Comment:
Shortcut for the future: you don't need to include the `content` tag if there's no inline elements.